### DISCUSSION TITLE
Relationship is not working in PIMCORE data objects

### DISCUSSION THREAD

**Discussion by anoopissac** [pimcore/pimcore#17170] [Q&A / Support] [answered]
We have created two data object classes let's say parent and child.
Child have many to one relationship to parent and its configured successfully in classes/general settings section of PIMCORE.
We created multiple parent objects.
While creating child objects, our intention was, select one of the parent objects from the drop-down box. But the parent object drop-down box is always empty, it's not populating any of the parent object.

<img width="385" alt="image" src="https://github.com/pimcore/pimcore/assets/20300506/5d372eed-8693-4321-9b82-6ba222f61cf4">

<img width="566" alt="image" src="https://github.com/pimcore/pimcore/assets/20300506/8e3aa188-0cba-49df-9a5c-7b6eff6f31bd">

in the screen shot consider "PPVendorDetails" is the parent object.

**Comment by markus-moser** [comment, marked as answer]
Maybe the problem is that the simple backend search is not indexed? Please try to execute `bin/console pimcore:search-backend-reindex` and see if it works then.

**Reply by anoopissac** [reply]
Its worked, Thanks a Lot
